### DISCUSSION TITLE
Getting properties of associations with py-capellambse

### DISCUSSION THREAD

**Discussion by khammoutene** [dbinfrago/py-capellambse#190] [Q&A] [answered]
Hello,

I'd like to know how I can retrieve the properties of an association between classes like:

- Name
- Kind (e.g. Aggregation)
- Role of each side of the association
- Min./Max. Card. of each side of the association

Thank you

**Comment by ewuerger** [comment, marked as answer]
Hi,

thank you for the report. Currently there are only properties in our `crosslayers` model definitions defined. I created an issue (#193) and already solved the defects you were referring to. After the linked PR (#194) is merged you can see the effects in our demo notebook in the documentation: [How Tos/03 Data Values](https://dsd-dbs.github.io/py-capellambse/examples/03%20Data%20Values.html).

Cheers.
